others/career-development.md

```diff
@@ -0,0 +1,73 @@
+# Career Development
+
+## CV
+
+### Best practice
+
+- Keep it short and concise, 1-2 pages maximum
+- Put your details at the top. Only these details are necessary:
+  - Name
+  - Email
+  - Phone number
+  - Your city (not your address since this is personal information! If they need it they'll ask for it later)
+- Use the following sections, and only populate them with relevant details starting from the most recent first
+  1. Education
+  2. Employment
+     - Use action verbs for each bullet point (see: <https://www.indeed.com/career-advice/resumes-cover-letters/action-verbs-to-make-your-resume-stand-out>)
+     - Show your accomplishments, with numbers if you can (e.g. Improved landing page time to interactive by 50%)
+  3. Skills
+  4. Personal projects
+  5. Personal interests (optional)
+
+### Resources
+
+Guide to writing a CV
+
+- <https://www.freecodecamp.org/news/writing-a-killer-software-engineering-resume-b11c91ef699d/>
+
+CV sample
+
+- <https://careercup.com/resume>
+
+## Interviews
+
+Interview questions with answers
+
+- <https://github.com/FAQGURU/FAQGURU>
+- <https://github.com/30-seconds/30-seconds-of-interviews>
+- <https://www.wisebread.com/how-to-answer-23-of-the-most-common-interview-questions>
+
+Questions to ask the company
+
+- <https://github.com/Twipped/InterviewThis>
+
+Quizzes
+
+- <https://davidshariff.com/quiz/>
+
+## Personal projects
```

Comment:
We're going to be working with them on portfolio pieces at some length, so we probably want to cover how they're going to be describing that on the CV rather than giving them advice on personal projects in general.